Sources/MapboxCoreNavigation/SDKVersionProvider.swift

```diff
@@ -0,0 +1,71 @@
+import Foundation
+
+struct VersionProvider {
```

Comment:
Could this be an extension on Bundle, where it would be more discoverable and potentially exposed to application code in the future if necessary? The Provider pattern isn’t really an idiomatic pattern in Swift.

Reply:
Done